### DISCUSSION TITLE
OpenJDK

### DISCUSSION THREAD

**Discussion by roggss** [nextgenhealthcare/connect#5798] [General]
Hi,

I removed Oracle Java and install OpenJDK

Do I need to reinstall Mirth?

If not, how do I configure Mirth to use the OpenJDK.
I ran the MC Manager and I got "No JVM could be found...."

I also define the EXE4J_JAVA_HOME on my system variables, still not working.

Please help.

Thanks

**Comment by pacmano1** [comment]
- Please always include the operating system in your posts.
- You did not say what installer you used for OpenJDK.    Please include that.
- You did not say what version of OpenJDK.   Please include that.

No, you do not need to reinstall Mirth.   

If you use Amazon Correto's OpenJDK release it usually fixes the paths and environment variables correctly.

**Reply by pacmano1** [reply]
If you use java 9 or later please _read_ the release notes on extra required options in mcserver.vmoptions and mcservice.vmoptions.

**Reply by roggss** [reply]
windows server 2012
Microsoft OpenJDK 8

I tried the Correto's OpenJDK and it went all right.

One thing i noticed though, the Mirth icon on the system tray is saying "Mirth Connect Service could not be started", but in fact it was started fine. Any idea?


![image](https://github.com/nextgenhealthcare/connect/assets/96202654/fe41b01b-9023-4992-a473-f7047a66db3e)

**Reply by pacmano1** [reply]
Not sure, I don't run Mirth on Windows.   Maybe check Event Viewer? I am also not sure if that is an artifact of your earlier failed install.

**Reply by roggss** [reply]
Could not find any in the Event Viewer.

The only difference I did on this Mirth Installation was i ran the Oracle-databse.sql manually since Mirth is not creating the tables after switch from derby (default) to Oracle.

**Reply by pacmano1** [reply]
I'd look at mirth.log.

**Reply by roggss** [reply]
I  dont see any error on the log.

**Reply by pacmano1** [reply]
If mirth.log shows something like this after your most recent restart I would not worry about it.    Obviously your DB and java is different and I am on Linux, but 

![image](https://github.com/nextgenhealthcare/connect/assets/44065187/ea3d7dbe-11ff-47ae-a202-b925a1529be0)

**Reply by roggss** [reply]
yes, exactly like that.

Okay thanks for that confirmation.
